src/site/shared/containers/ItemNav/index.tsx

```diff
@@ -122,6 +132,15 @@ export const ItemNav = <D,>({ info, config, additionalData,
         return () => DragDropHandlers.removeListener(resetListener);
     }, [setState]);
 
+    // Keep track of if item is being placed
+    useEffect(() => {
+        if (curItemID) {
+            setIsPlacing();
+        }
+        else {
+            setIsPlacing();
+        }
```

Comment:
I may be missing something, but is this if else statement necessary? It is doing the same thing regardless of the condition

Reply:
No that does not do anything, I definitely just missed it before committing; it should be fixed now!